src/ripple/app/tests/Regression_test.cpp

```diff
@@ -95,8 +94,7 @@ struct Regression_test : public beast::unit_test::suite
 
             auto const result = ripple::apply(env.app(),
```

Comment:
#1328 is a prerequisite for removing `config` from `apply`, so I'll get back to this one.